### ISSUE TITLE
React-router-redux "Module Not Found" Even after clean install

### ISSUE THREAD

**Issue by sonarforte** [ReactTraining/react-router#5230]
## Version
5.0.0-alpha.6

## Steps to reproduce

I'm trying to upgrade to the new version of React Router.

I installed the latest version of react-router-redux. Here is my package.json:

```javascript
{
  "name": "web_client",
  "version": "0.1.0",
  "private": true,
  "homepage": "http://app.usefeed.me",
  "devDependencies": {
    "react-scripts": "^0.9.0",
    "redux-devtools-extension": "^2.13.2"
  },
  "dependencies": {
    "babel-polyfill": "^6.23.0",
    "classnames": "^2.2.5",
    "crypto": "0.0.3",
    "dns": "^0.2.2",
    "history": "^4.6.1",
    "isomorphic-fetch": "^2.2.1",
    "jsonwebtoken": "^7.4.1",
    "jws": "^3.1.4",
    "jwt-simple": "^0.5.1",
    "prop-types": "^15.5.10",
    "react": "^15.5.4",
    "react-dom": "^15.5.4",
    "react-redux": "^5.0.5",
    "react-redux-form": "^1.12.0",
    "react-router-dom": "^4.1.1",
    "react-router-redux": "^5.0.0-alpha.6",
    "redux": "^3.6.0",
    "redux-logger": "^2.10.2",
    "redux-saga": "^0.14.8",
    "semantic-ui-css": "^2.2.10",
    "semantic-ui-react": "^0.67.2",
    "validator": "^7.0.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test --env=jsdom",
    "eject": "react-scripts eject"
  }
}
```

and entry point for my app:

```javascript
import 'babel-polyfill'
import React from 'react'
import ReactDOM from 'react-dom'
import { combineReducers, createStore, applyMiddleware, } from 'redux'
import { Provider } from 'react-redux'
import { Route, } from 'react-router-dom'
import createHistory from 'history/createBrowserHistory';
import { 
  ConnectedRouter,
  routerReducer,
  routerMiddleware as createRouterMiddleware,
} from 'react-router-redux'

import createLogger from 'redux-logger'
import createSagaMiddleware from 'redux-saga'  
import { composeWithDevTools } from 'redux-devtools-extension';

import reducers from './reducers/'
import IndexSagas from './index-sagas'
import App from './containers/App'

const history = createHistory()
const sagaMiddleware = createSagaMiddleware()
const loggerMiddleware = createLogger()
const routerMiddleware = createRouterMiddleware(history)

/* eslint-disable no-underscore-dangle */
const store = createStore(
  combineReducers({
    ...reducers,
    router: routerReducer,
  }),
  composeWithDevTools(
    applyMiddleware(
      sagaMiddleware,
      routerMiddleware,
      loggerMiddleware, 
    )
  ),
)
/* eslint-enable */

sagaMiddleware.run(IndexSagas)

ReactDOM.render(
  <Provider store={store}>
    <ConnectedRouter history={history}>
      <Route path="/" component={App} />
    </ConnectedRouter>
  </Provider>,
  document.getElementById('root')
);
```

## Expected Behavior

I thought this would work, as it is very similar to the example in the repo's Readme.

## Actual Behavior

However, I get these error messages in webpack:

`Uncaught Error: Cannot find module "react-router-redux"`

> Module not found: [CaseSensitivePathsPlugin] `/Users/steven/feedme/web_client/node_modules/react-router-redux/lib/index.js` does not match the corresponding path on disk - File does not exist.

This is after I removed `node_modules` and did a clean install.

Any suggestions here? Thanks.

**Comment by timdorr** [comment]
This has something to do with your bundler config (webpack, it looks like), not the library. Nothing in the package.json points to a `lib/index.js` file and we don't have that file in the bundle, so it shouldn't be looking there.

**Comment by sonarforte** [comment]
Thanks, I fixed this by upgrading `create-react-app`.

**Comment by smilexu** [comment]
@sonarforte  I meet same problem, how do you fix that? I use quite latest version of create react app.

**Comment by sonarforte** [comment]
@smilexu My best advice is to delete `node_modules`, reinstall them, upgrade `create-react-app`, and that should work-- if it doesn't, maybe open a new issue because I'm not sure why that would happen.

**Comment by xgzl87** [comment]
I meet same problem, how do you fix that? I use quite latest version of create react app.
use typescript

**Comment by Altiano** [comment]
hi @sonarforte, could you five me example repo like your config (using saga etc) because I am going for it too.

**Comment by xgzl87** [comment]
fixed add npm i @types/react-router-redux

**Comment by nharrisanalyst** [comment]
if you get this error it means that you are not using version 5.0 of react-router-redux

I don't know why but for some reason some thought that npm should install v4 of create react redux to install you have to manually install the latest version 

$ npm install --save react-router-redux@next

**Comment by jcicero518** [comment]
I'm having a similar issue, but i'm not using create-react-app. If this helps anyone, react-router-redux@5.0.0-alpha.6 says that it requires React@15.6.2. I was getting an unmet peer dependency when trying to install react-router-redux. Now that I've updated React itself I'm getting:

`ERROR in ./node_modules/react-router-redux/lib/index.js`
`Module build failed: Error: ENOENT: no such file or directory`

I noticed @sonarforte was using React@15.5.4 as I was. Thing is, there is no lib directory in react-router-redux@5.x .. what gives?

Some of these semvers may be out of whack since I just updated React & ReactDOM but here's the notable dependencies:

"react": "^15.6.2",
    "react-dom": "^15.6.2",
    "react-icons": "^2.2.7",
    "react-redux": "^5.0.7",
    "react-router": "^4.1.2",
    "react-router-dom": "^4.1.2",
    "react-router-redux": "5.0.0-alpha.6",
    "react-spinkit": "^3.0.0",
    "react-transition-group": "^1.2.1",
    "redux": "^3.7.2",
    "redux-immutable": "^4.0.0",
    "redux-saga": "^0.16.0"
